### PR TITLE
CB-13502 - FreeIPA OS disk remains 30 GB

### PR DESCRIPTION
**Pull request by pdarvasi**
Tested with Azure FreeIPA:

```
[cloudbreak@ipaserver0 ~]$ lsblk
NAME   MAJ:MIN RM  SIZE RO TYPE MOUNTPOINT
fd0      2:0    1    4K  0 disk
sda      8:0    0  100G  0 disk
|-sda1   8:1    0  500M  0 part /boot
`-sda2   8:2    0 99.5G  0 part /
sdb      8:16   0  200G  0 disk
`-sdb1   8:17   0  200G  0 part /mnt/resource
sr0     11:0    1  638K  0 rom
```

### CODE REVIEW

**Review by lacikaaa — APPROVED** [review]
LGTM, wait for @gergopapi2 review please before merging

**Review by gergopapi2 — APPROVED** [review]

**Pull request merged**
